site/src/main/java/uk/nhs/digital/ps/beans/Publication.java

```diff
@@ -21,13 +22,17 @@
 
     private RestrictableDate nominalPublicationDate;
 
+    public static Publication getPublicationInFolder(HippoFolder folder) {
```

Comment:
Is it worth renaming to "getPublicationContent", as otherwise it looks like it might be a generic search type utility?

Reply:
It's more internal jargon, the "Folder" bit is important to me. It's the logic for finding  publication in a given folder. 

something to consider:
* `Publication.getPublicationFromFolder(folder)`
* `Publication.getPublicationInFolder(folder)`